.github/workflows/pull-request.yml

```
@@ -223,10 +223,32 @@ jobs:
+      - name: Import Sample Data into Database
+        run: docker compose exec api sh -c "yes yes | pnpm import:sample-data"
+      - name: Validate Database Records
+        run: docker compose exec api pnpm test:db-connection
+      - name: Stop Services
+        run: docker compose down
+
   Test-Docusaurus-Deployment:
     name: Test Deployment to https://docs-api.talawa.io
     runs-on: ubuntu-latest
-    needs: [Run-Tests]
+    needs: [Run-Tests,Database-Access-Check]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)